lhotse/serialization.py

```diff
@@ -815,6 +815,82 @@ def handles_special_case(self, identifier: Pathlike) -> bool:
     def is_applicable(self, identifier: Pathlike) -> bool:
         return is_valid_url(identifier)
 
+    
+@lru_cache(1)
+def get_lhotse_msc_override_protocols() -> Any:
+    return os.getenv("LHOTSE_MSC_OVERRIDE_PROTOCOLS", None)
```

Comment:
Please document these environment variables in Lhotse's top-level `README.md` where it lists all env vars used to modify lhotse behavior.